### DISCUSSION TITLE
Flax farm no harvesters publishing plots to farmer

### DISCUSSION THREAD

**Discussion by BasvanH** [guydavis/machinaris#205] [Q&A] [answered]
Hello,

On my full node I do not see my flax harvesters listed.

```
Farming status: Farming
Total flax farmed: 0.0
User transaction fees: 0.0
Block rewards: 0.0
Last height farmed: 0
Plot count: 0
Total size of plots: 0.000 GiB
Estimated network space: 2438.282 PiB
Expected time to win: Unknown
Note: log into your key using 'flax wallet show' to see rewards for each key
```

At one of my harvesters:
```
021-07-26T09:09:16.530  flax.plotting.check_plots        : INFO     Summary
2021-07-26T09:09:16.530  flax.plotting.check_plots        : INFO     Found 18 valid plots, total size 1.78194 TiB
2021-07-26T09:09:16.531  flax.plotting.check_plots        : INFO     18 plots of size 32
```

Certs are all in place and valid.
Ports open at flax farmer: -p 8444:8444 -p 8447:8447 -p 6888:6888 -p 6885:6885 -p 8926:8926 -p 8927:8927
Ports open at flax harvester: -p 8926:8926 -p 8927:8927
Farmer IP is set in harvester config.yaml
Restarted everything.
There are no errors in flaxdog of harvesters.

I'm out of idea's, what could be the cause of the flax harvesters not publishing their plots to the farmer? Any help is more than wolcome!

Kind regards,
Bastiaan

**Comment by guydavis** [comment, marked as answer]
Sorry about this confusion.  To be clear, you are farming Flax successfully on your harvester(s).

Currently Flax (a fork) is based upon the Chia1.1.7 binaries, while Chia itself is at version 1.2.2.  Due to this Flax reports like the old Chia binaries used to do, without any harvester info.  Chia only introduced the per-harvester details in `chia farm summary` in version 1.2.0+.

Example on my current LAN, as executed from my Machinaris controller: 
![image](https://user-images.githubusercontent.com/12936/127004140-fc8e3d18-75c0-4497-ab1c-de0d688fc532.png)

That said, while Flax on the fullnode only reports local plots, Machinaris (through it's Worker mechanism) will correctly summarize plot counts and actually re-calculates a LAN-wide Flax estimated-time-to-win.  

![image](https://user-images.githubusercontent.com/12936/127004308-63667c32-b459-4798-ac53-297fa423b453.png)

Flax devs are [working on supporting NFT pooling](https://discord.com/channels/842422792593866782/849382899101859870/868264765447868457).  As soon as they provide a new version, Machinaris will bundle it.

Hope this helps!  If so, please mark this Discussion as Answered.

**Reply by BasvanH** [reply]
Thats the exact information I was looking for, you have answered my question. 

Thank you very much @guydavis
